### PR TITLE
cope

### PR DESCRIPTION
**Pull request by true-binar-White-Rabbit**
[DEMO LINK](https://true-binar-White-Rabbit.github.io/react_autocomplete/) 

Hi, everyone!
Please kindly request! 
Before talking about the requirement to pass automatic tests, check out the proposed variant. 
Review the suggested code and the approach to solving the original task. 
Suggest code optimization options and formulate the shortcomings of the proposed decision.
Ty!

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,16 +1,60 @@
 import React from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [query, setQuery] = React.useState('');
+  const [isDropedDown, setIsDropedDown] = React.useState(false);
+  const [selectedPerson, setSelectedPerson] = React.useState<Person | null>(
```
Comment (anastasiiavorobiova) [inline src/App.tsx:9]:
```suggestion
  const [query, setQuery] = useState('');
  const [isDropedDown, setIsDropedDown] = useState(false);
  const [selectedPerson, setSelectedPerson] = useState<Person | null>(
```

File: src/App.tsx [hunk 2]
```diff
@@ -1,16 +1,60 @@
 import React from 'react';
```
Comment (anastasiiavorobiova) [inline src/App.tsx:1]:
```suggestion
import React, { useState } from 'react';
```

File: src/App.tsx [hunk 3]
```diff
@@ -19,55 +63,76 @@ export const App: React.FC = () => {
...
+              is-danger
+              is-light
+              mt-3
+              is-align-self-flex-start"
+            role="alert"
+            data-cy="no-suggestions-message"
+          >
+            <p className="has-text-danger">No matching suggestions</p>
+          </div>
+        ) : selectedPerson && getFilterPeople().length > 0 ? (
```
Comment (anastasiiavorobiova) [inline src/App.tsx:122]: Avoid using nested ternary

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great start! Keep doing! Feel free to ask for help in the chat to pass tests

File: src/App.tsx (hunk 1 above)
Comment (true-binar-White-Rabbit) [reply src/App.tsx:9]: ty, but this is a matter of taste, not rationality.

File: src/App.tsx (hunk 3 above)
Comment (true-binar-White-Rabbit) [reply src/App.tsx:122]: sorry, but this is not nested ternary, i use the only one ternary expression that compares two conditions via the operator ?: to determine which JSX block to render

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
If you feel some of the comments are unclear, please ask for help in the chat or come to the Q&A session. We will deal with everything)

File: src/App.tsx (hunk 3 above)
Comment (maxim2310) [reply src/App.tsx:122]: yes it's actually called nested ternary, try to rewrite without this antipattern

File: src/App.tsx (hunk 1 above)
Comment (maxim2310) [reply src/App.tsx:9]: But would be better stick to react doc

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Good work, but lets make some improvement 
try to figure out with tests 
image

**Comment by true-binar-White-Rabbit** [comment]
> Good work, but lets make some improvement try to figure out with tests image

is my title not updated after a person is selected? or when you delete the name, the title "No selected person" does not appear?

**Pull request closed without merging**